saleor/graphql/checkout/mutations.py

```diff
@@ -315,8 +335,8 @@ def perform_mutation(cls, _root, info, **data):
             checkout = models.Checkout(user=user)
         else:
             checkout = models.Checkout()
-
         cleaned_input = cls.clean_input(info, checkout, data.get("input"))
+        checkout.channel = cleaned_input["channel"]
```

Comment:
I think this should happen in `clean_input`. We either use the value passed in the input or use the default from `info.context`.